_config/config.yml

```diff
@@ -14,5 +14,6 @@ CMSPageSettingsController:
   extensions:
     - ContentReviewCMSExtension
 SiteConfig:
+  content_review_gridfield_threshold: 2000
```

Comment:
I don't think we need this now that the private static is set on `ContentReviewDefaultSettings`